src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -442,24 +466,28 @@ class SpanDetail extends React.Component<Props, State> {
                   : null}
               </Row>
               <Tags span={span} />
-              {map(span?.data ?? {}, (v, k) => {
-                const key = (k as unknown) as string;
-                const value = (v as unknown) as number;
-                return (
-                  <Row title={key} key={key}>
-                    {SIZE_DATA.has(key) ? (
-                      <React.Fragment>
-                        <FileSize bytes={value} />
-                        {value >= 1024 && (
-                          <span>{` (${JSON.stringify(value, null, 4) || ''} B)`}</span>
-                        )}
-                      </React.Fragment>
-                    ) : (
-                      JSON.stringify(value, null, 4) || ''
+              {allZeroSizes && (
+                <TextTr>
+                  The following sizes were not collected for security reasons. Check if
+                  this host serves <span className="val-string">timing-allow-origin</span>
+                  for your domain. You may have to enable this collection manually.
```

Comment:
The response header doesn't have to be enabled for the domain. It can also just be enabled for the relevant endpoint path.

Reply:
How about we update the copy to this?

> The following sizes were not collected for security reasons. Check if the host serves the appropriate `Timing-Allow-Origin` header. You may have to enable this collection manually.

Reply:
I think that works 👍

Reply:
Updated in 6de3e5d